paasta_tools/check_chronos_jobs.py

```diff
@@ -264,6 +277,19 @@ def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster
     return output, sensu_status
 
 
+def job_expected_runtime_callback(client, chronos_job):
```

Comment:
We don't always need to know 99thPercentile of the job.
But the only way to know whether to make this request or not, is to verify if last_run_time + interval is bigger than datetime.now in job_is_stuck().

If I make this a "normal" function that is called from job_is_stuck() and returns int, then I'll have to pass 'client' and 'chronos_job' as arguments to sensu_message_status_for_jobs(), then to job_is_stuck().

With the current functional approach I only pass one argument.
In my opinion passing 'expected_runtime_callback' is more straightforward way than passing  'client' and 'chronos_job' and expect future developers to understand what are they and how/where do we use them.